src/gc/collector.cpp

```diff
@@ -48,6 +48,31 @@ namespace gc {
 FILE* trace_fp;
 #endif
 
+unsigned classesAllocatedSinceCollection = 0;
```

Comment:
seems like the classesAllocatedSinceCollection stuff snuck into this PR?  I can't see a use of it in this change.